File: core/src/main/java/com/bing/lan/core/base/mapper/UserinfoMapper.java
package com.bing.lan.core.base.mapper;

import com.bing.lan.core.base.domain.Userinfo;

import java.util.List;

public interface UserinfoMapper {

    int deleteByPrimaryKey(Long id);

    int insert(Userinfo record);

    Userinfo selectByPrimaryKey(Long id);

    List<Userinfo> selectAll();

    int updateByPrimaryKey(Userinfo record);
}

File: core/src/main/kotlin/com/bing/lan/core/base/mapper/UserinfoMapper.kt
package com.bing.lan.core.base.mapper

import com.bing.lan.core.base.domain.Userinfo

interface UserinfoMapper {

    fun deleteByPrimaryKey(id: Long?): Int

    fun insert(record: Userinfo): Int

    fun selectByPrimaryKey(id: Long?): Userinfo

    fun selectAll(): List<Userinfo>

    fun updateByPrimaryKey(record: Userinfo): Int
}